app/src/main/java/ch/epfl/sdp/blindly/recording/AudioLibraryAdapter.kt

```diff
@@ -143,13 +190,47 @@ class AudioLibraryAdapter(var recordList: ArrayList<AudioRecord>,
         }
     }
 
+    private fun bindSeekBarNavigation(playBar: SeekBar,
+                                      playTimer: Chronometer,
+                                      remainingTimer: Chronometer,
+                                      playPauseButton: AppCompatImageButton,
+                                      playBarThread: Runnable,
+                                      position: Int) {
+        playBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
+            override fun onProgressChanged(playBar: SeekBar,
+                                           progress: Int, fromUser: Boolean) {
+            }
+
+            override fun onStartTrackingTouch(playBar: SeekBar) {
+                /*if (isPlayerStopped) {
+                    resetRecordPlayer(position, playTimer, remainingTimer, playPauseButton, playBar)
+                } else {
+                    mediaPlayer?.pause()
+                    setStoppedView(playTimer, remainingTimer, playPauseButton, true)
+                }*/
```

Comment:
Why is that section commented ? Is it buggy ? Or should we remove it ?

Reply:
I'm currently working on it !

Reply:
my bad :P